catalog-spec/catalog-spec.md

```diff
@@ -120,15 +120,15 @@ details on the two types and how you might use them see the [Static and Dynamic
 | id           | string        | **REQUIRED.** Identifier for the catalog.                    |
 | title        | string        | A short descriptive one-line title for the catalog.          |
 | description  | string        | **REQUIRED.** Detailed multi-line description to fully explain the catalog. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
-| summaries    | Map<string, [*]\|[Range Object](#range-object)> | A map of property summaries, either a set of values or a range. |
+| summaries    | Map<string, [*]\|[Stats Object](#stats-object)> | A map of property summaries, either a set of values or a statistics such as a range. |
```

Comment:
Super minor grammatical point, but I think it should be 'a statistic' instead of 'a statistics'

Reply:
Thanks for spotting this typo.